conf/broker.conf

```diff
@@ -557,6 +557,9 @@ zookeeperSessionExpiredPolicy=reconnect
 # Enable or disable system topic
 systemTopicEnabled=true
 
+# Deploy the schema compatibility checker for a specific schema type to enforce schema compatibility check
+schemaRegistryCompatibilityCheckers=org.apache.pulsar.broker.service.schema.JsonSchemaCompatibilityCheck,org.apache.pulsar.broker.service.schema.AvroSchemaCompatibilityCheck,org.apache.pulsar.broker.service.schema.ProtobufNativeSchemaCompatibilityCheck
```

Comment:
I re-run the check again just now.

I checked the failure check yesterday and guess it might be caused by the overlength. 
As we discussed w/ engineers just now, can you try to split into multiple lines if the check fails again?

Ignore this comment and ping me to merge if the check passes.

Reply:
The test has been completed successfully :)